From d0644a43a31b9401616b739cd03065faf16c9e1c Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Mon, 11 Jun 2018 15:53:32 +0800
Subject: [PATCH 3961/5242] MLK-18578: hdmi arc: fix noise issue with
 96kHz/192kHz.

commit  e06411b7aded522f7007345edcb1190a36c18458 from
https://source.codeaurora.org/external/imx/linux-imx.git

After enable the boost circuit of tx_ana_ctrl_reg_5, the noise
issue with ARC is gone.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Sandor.yu <sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-arc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-arc.c b/drivers/gpu/drm/imx/hdp/imx-arc.c
index b6774c5..be94437 100644
--- a/drivers/gpu/drm/imx/hdp/imx-arc.c
+++ b/drivers/gpu/drm/imx/hdp/imx-arc.c
@@ -87,7 +87,7 @@ void imx_arc_config(state_struct *state)
 	/* register TX_ANA_CTRL_REG_2 */
 	Afe_write(state, 0x5021, 0x030C);
 	/* register TX_ANA_CTRL_REG_5 */
-	Afe_write(state, 0x5029, 0x0000);
+	Afe_write(state, 0x5029, 0x0010);
 	udelay(100);
 	/* register TX_ANA_CTRL_REG_4 */
 	Afe_write(state, 0x5027, 0x4001);
-- 
1.7.9.5

